builtin_interfaces/Time stamp
float64 latitude
float64 longitude
float64 altitude
float64 reference_latitude
float64 reference_longitude
float64 reference_altitude
float64 reference_heading
DataXYZ velocity
DataXYZ covariance
uint8 covariance_type
uint16 service
int8 status
